tests/drivers/i2c/i2c_fram/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+tests:
+  drivers.i2c.api_fram:
+    depends_on: i2c
+    tags:
+      - drivers
+      - i2c
+    filter: dt_alias_exists("mb85")
```

Comment:
we need add harness requirement for click_mb85

Reply:
I think leaving the fixture as mb85 is ok, there *are* other breakouts, the key here is that the board has a i2c bus alias labeled mb85, that could be the i2c bus connected to a particular click board socket.

Speaking of click boards, it seems like we might want an added alias for click_i2cX as there's the strong potential, e.g. the nucleo 144 shield, where more than one i2c is possible.

I don't know the best route to deal with multiple shield boards though and multiple shield board sockets, maybe @jfischer-no would have some helpful hints on that front.

Reply:
may just put `arduino`  will be ok

Reply:
@teburd sorry, not sure I understand, maybe `boards/shields/arduino_uno_click` is an example of what you are looking for.

Reply:
I'll take a closer look at that, thank you